src/app/reducers/service-offerings/redux/service-offerings.effects.ts

```diff
@@ -53,6 +50,15 @@ export class ServiceOfferingEffects {
       );
     });
 
+  @Effect()
+  loadCompatibilityPolicy$: Observable<Action> = this.actions$
```

Comment:
is it required to fetch them using effects? let's change the logic not to store policies in state